pkgs/misc/emulators/citra/base.nix

```
@@ -0,0 +1,71 @@
+{ pname, version, src, branchDesc, branchName
+, lib, stdenv, config, fetchFromGitHub
+, catch2, cmake, ninja, pkg-config, wrapQtAppsHook
+, boost173, ffmpeg, libusb1, qtbase, qtdeclarative, qtmultimedia, qttools, rapidjson, SDL2
+, alsaSupport      ? stdenv.isLinux,                      alsa-lib
+, discordSupport   ? true
+, fdkSupport       ? true,                                fdk_aac
+, jackaudioSupport ? true,                                libjack2
+, onlineSupport    ? true
+, pulseSupport     ? config.pulseaudio or stdenv.isLinux, libpulseaudio
+, sndioSupport     ? true,                                sndio
+, udevSupport      ? stdenv.isLinux,                      udev
+}:
+
+with lib;
+let compat-src = ./compatibility_list.json; # Updated 1/12/22 from https://api.citra-emu.org/gamedb/
+in stdenv.mkDerivation {
+  inherit pname version src;
+  cmakeFlags = [
```

This feels a bit crammed to me
```suggestion
  inherit pname version src;
  
  cmakeFlags = [
```